presto-main/src/main/java/com/facebook/presto/operator/scalar/HmacFunctions.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+import com.google.common.hash.Hashing;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+
+/**
+ * Functions to compute HMAC.
+ */
+public final class HmacFunctions
+{
+    private HmacFunctions() {}
+
+    @Description("compute HMAC MD5")
```

Comment:
`compute` -> `Compute`. Please update the other function descriptions too.

Reply:
`Compute HMAC with MD5` or `Compute MD5 HMAC` (Please update other functions too)